### DISCUSSION TITLE
Can bevy be added to this project?

### DISCUSSION THREAD

**Discussion by lo-ferris** [hannobraun/fornjot#1376] [General]

**Comment by hannobraun** [comment]
It's possible, but I'm not convinced it would be a good idea.

Converting this issue to a discussion, since it doesn't track any kind of planned work.

**Reply by lo-ferris** [reply]
Thank you for your reply

**Comment by XYCaptain** [comment]
I think it's a good idea！

**Reply by hannobraun** [reply]
Why though? How exactly would it be used? Who would do the work? And what would the benefit be?

**Reply by XYCaptain** [reply]
I expect to learn a lot from doing this. Validate a lot of my own ideas and maybe make the kernel have a better interactive experience.

**Reply by hannobraun** [reply]
Ah, so you're saying you want to do it? Good luck! I'm looking forward to see what you come up with!

**Reply by XYCaptain** [reply]
Yup. I just started learning rust and your project is worth learning. I also have some development experience of 3d modeling and 3d rendering. Thanks for your work and giving us the opportunity to communication.
